.github/workflows/ci.yaml

```diff
@@ -52,46 +52,6 @@ jobs:
           npm install
           npm run build
 
-      - name: Bring in rubin_chart repo
-        uses: actions/checkout@v4
-        with:
-          repository: 'lsst-sitcom/rubin_chart'
-          path: rubin_chart
-          ref: ${{ env.DDV_BRANCH }}
-
-      - name: Bring in DDV Flutter app repo
-        uses: actions/checkout@v4
-        with: 
-          repository: 'lsst-ts/rubintv_visualization'
-          path: ddv
-          ref: ${{ env.DDV_BRANCH }}
-
-      - name: Set up Flutter environment
-        uses: subosito/flutter-action@v2
-        with:
-          channel: stable
-
-      - name: Set up .env file
-        # TODO: Make ADDRESS configurable at runtime.
-        # the first part could be 'rubintv' or 'rubintv-dev'
-        # see DM-44660 https://rubinobs.atlassian.net/browse/DM-44660
-        run: echo -e "ADDRESS=rubintv/ws/ddv" > ${{ github.workspace }}/ddv/.env
```

Comment:
I think the `scripts/build-flutter-app.sh` is missing the `ADDRESS` environment variable.